sources/packages/backend/apps/api/src/route-controllers/student-loan-balance/student-loan-balance.aest.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+} from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { StudentService } from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+import BaseController from "../BaseController";
+import { StudentLoanBalanceAPIOutDTO } from "./models/student-loan-balance.dto";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("student-loan-balance")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-student-loan-balance`)
+export class StudentLoanBalanceAESTController extends BaseController {
+  constructor(private readonly studentService: StudentService) {
+    super();
+  }
+
+  /**
+   * Get the monthly part-time loan balance details of
+   * the given student upto 12 months.
+   * @param studentId student.
+   * @returns student monthly part-time loan balance.
+   */
+  @ApiNotFoundResponse({
+    description: "Student not found.",
+  })
+  @Get("part-time/student/:studentId")
```

Comment:
I am not sure about using the `part-time` at this route and at this level. Maybe it can be a future filter but right now we are not differentiating anywhere else.

Reply:
I added the `part-time` to the API (and also to the component) as we are precisely dealing with part-time data here.

I see the point of API being more generic with respect to student loan balance. But from the UI perspective the Vue component is about part time loan balance and I prefer that to be named with part time. Let me know your thoughts.

Reply:
I am ok with the way you described 😉